sdk-base/build.gradle.kts

```diff
@@ -28,8 +28,20 @@ dependencies {
     api(project(":maps-core"))
     api(project(":common"))
   } else {
-    api(Dependencies.mapboxGlNative)
-    api(Dependencies.mapboxCoreCommon)
+    api(Dependencies.mapboxGlNative) {
+      exclude("androidx.test.ext", "junit")
+      exclude("androidx.test", "rules")
+      exclude("androidx.test", "runner")
+      exclude("androidx.test", "core")
+      exclude("androidx.test", "monitor")
+    }
+    api(Dependencies.mapboxCoreCommon) {
+      exclude("androidx.test.ext", "junit")
+      exclude("androidx.test", "rules")
+      exclude("androidx.test", "runner")
+      exclude("androidx.test", "core")
+      exclude("androidx.test", "monitor")
+    }
```

Comment:
can we validate maybe in the dependency graph that these excluded modules doesn't show up as SDK dependency?

Reply:
that's still a temp fix that we will not release, we will check it properly with upcoming beta.2

Reply:
Is the beta.2 coming later this week or early next week before we cut the beta.1 Android SDK release?

Reply:
From what I see we can't release with beta.1 as it may break user's testing code and even excluding things here does not sound for me as a reliable plan. I'd aim for beta.2 and removing those excludes but we would not make it - we'll test dependency graph before our actual release again.

Reply:
Pom of the sdk will contain the exclusion sections, e.g.

```
    <dependency>
      <groupId>com.mapbox.maps</groupId>
      <artifactId>android-core</artifactId>
      <version>10.6.0-beta.1</version>
      <scope>compile</scope>
      <exclusions>
        <exclusion>
          <artifactId>runner</artifactId>
          <groupId>androidx.test</groupId>
        </exclusion>
        <exclusion>
          <artifactId>core</artifactId>
          <groupId>androidx.test</groupId>
        </exclusion>
        <exclusion>
          <artifactId>junit</artifactId>
          <groupId>androidx.test.ext</groupId>
        </exclusion>
        <exclusion>
          <artifactId>rules</artifactId>
          <groupId>androidx.test</groupId>
        </exclusion>
        <exclusion>
          <artifactId>monitor</artifactId>
          <groupId>androidx.test</groupId>
        </exclusion>
      </exclusions>
    </dependency>
```

Suppose consumers of maps should correctly handle them afterwards, but will be nice to check using snapshot ofc